.eslintrc.json

```diff
@@ -5,6 +5,7 @@
     "no-param-reassign": 0,
     "no-duplicate-imports": 0,
     "no-underscore-dangle": 0,
+    "import/no-duplicates": 0,
```

Comment:
Unless this is being used _everywhere_, this should be done as a comment where it is necessary.